test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -173,36 +207,57 @@ void test_ucp_ep_reconfig::entity::store_config()
     }
 
     m_cfg_index = ep()->cfg_index;
+
+    /* Calculate number of reused lanes */
+    auto num_reused    = (ucp_ep_num_lanes(ep()) / 2) / num_paths();

```

Comment:
seems there is an assumption here related to the behavior of num_paths in IB transports. But the test must be generic and not make such assumptions for specific transprots

Reply:
can you clarify what is the assumption?